integration/gotests/gotest_test.go

```diff
@@ -95,6 +95,9 @@ func TestGoTest(t *testing.T) {
 		QEMUOpts: qemu.Options{
 			Timeout: 120 * time.Second,
 			Devices: []qemu.Device{
+				// Add BMC emulation for integration tests of pkg/ipmi
+				qemu.ArbitraryArgs{"-device", "ipmi-bmc-sim,id=bmc0"},
+				qemu.ArbitraryArgs{"-device", "isa-ipmi-kcs,bmc=bmc0,irq=5"},
```

Comment:
Does /dev/ipmi now show up in qemu?

Reply:
Yet to decide. Docker and qemu giving me a hard time setting this stuff up manually.

Reply:
Cant figure out. Build qemu 6.0 locally, build kernel 5.10 locally, created uroot locally, started qemu with the bmc and ipmi args.

Drops me into uroot shell.

`ls /dev` doesn't show /dev/ipmi nor /dev/bmc0

Reply:
Let's make sure it shows up before we push this changes to prevent too much back-n-forth.

Reply:
Try adding these one-at-a-time to see if it adds the ipmi device:

```
CONFIG_IPMI_HANDLER=y
CONFIG_IPMI_DEVICE_INTERFACE=y
CONFIG_IPMI_SI=y
CONFIG_IPMI_SSIF=y
CONFIG_DMI=y
CONFIG_IPMI_DMI_DECODE=y
CONFIG_IPMI_PLAT_DATA=y
```

And try adding the following to see if it adds the watchdog device:

```
CONFIG_IPMI_WATCHDOG=y
```